app/src/main/java/com/sdp/movemeet/models/FirebaseObject.java

```diff
@@ -4,6 +4,7 @@
  * An object that can be found in the Firebase Realtime Database/Firestore backend
  */
 public interface FirebaseObject {
```

Comment:
Why is this an interface? You should pull the abstract class version from the main

Reply:
Yes you're right. My branch works alright with this FirebaseObject implemented as an interface, but since you re-implemented as an abstract class in your pull-request this week, I will pull this version from the main to properly merge my code!